app/src/org/commcare/network/CommcareRequestGenerator.java

```diff
@@ -107,6 +108,11 @@ public static CommcareRequestGenerator buildNoAuthGenerator() {
             if (digest != null) {
                 params.put("state", "ccsh:" + digest);
             }
+
+            int getDaysSinceSync = SyncDetailCalculations.getDaysSinceLastSync();

```

Comment:
Maybe use a header instead ? If not, just wanna double check since we are adding a new parameter, it won't result in any cache busting on HQ ?

Reply:
this won't impact caching. We did discuss a header vs a URL param. I don't think it makes any difference in terms of implementing this feature. We did think that it could be useful to have in the logs (headers don't appear in the logs) though we do also get this data from the heartbeat requests if we wanted to do more analysis

Reply:
sounds good. Thanks!